features/step_definitions/subscription_steps.rb

```diff
@@ -4,13 +4,19 @@
   end
 end
 
+Given /subscription vouchers for vouchertypes? (.*)/ do |voucher_types|
+  voucher_types.split(/\s*,\s*/).each do |voucher_type|
+    create(:bundle, :name => voucher_type, :subscription => true, :fulfillment_needed => true)
+  end
+end
+
 Given /the following subscribers exist:/ do |subscribers|
   subscribers.hashes.each do |customer|
     first,last = customer['customer'].split(/\s+/)
     cust = find_customer(first,last) || create(:customer, :first_name => first, :last_name => last)
     customer['quantity'] ||= '1'
-    customer['subscriptions'].split(/\s*,\s*/).each do |season|
-      buy!(cust, Vouchertype.find_by(:name => season), customer['quantity'].to_i)
+    customer['subscriptions'].split(/\s*,\s*/).each do |subscription_type|
+      buy!(cust, Vouchertype.find_by(:name => subscription_type), customer['quantity'].to_i)
```

Comment:
are you certain this doesn't break other tests that rely on the current step definition??  this step is used by other scenarios, do they all pass locally?

Reply:
We just double-checked by running all cucumber local tests, and they all passed
224 scenarios (224 passed)
1810 steps (1810 passed)
7m33.514s